x-pack/plugins/fleet/server/routes/setup/index.ts

```diff
@@ -40,11 +78,25 @@ export const registerCreateFleetSetupRoute = (router: FleetAuthzRouter) => {
       fleetAuthz: {
         fleet: { setup: true },
       },
+      description: `Initiate agent setup`,
+      options: {
+        tags: ['Elastic Agents'],
```

Comment:
For now I copied the tags from the existing openapi spec, we could come up with more consistent tags.